index.js

```diff
@@ -3,30 +3,108 @@
  * @param {number[]} rolls - An array of numbers representing rolls on a die.
  * @returns {boolean} Returns `true` if all values in the array are numbers. Otherwise, return `false`.
  */
-function isValid(rolls) {}
+function isValid(rolls) {
+  let count = 0               //accumulator to add values
+   if (rolls.length === 0) return true 
+  // this is to check if rolls is [] but no need because for loop executes and nothing it will automatically return empty []
+
+  for(let i = 0; i < rolls.length; i++){
+    if (typeof rolls[i] === "number"){
+      count++
+    }
+  }
+  return (count < rolls.length  ?  false : true)
+  }
 
 /**
  * Finds a value in an array. If that value is in the array, returns it. Otherwise, returns `null`.
  * @param {number[]} rolls - An array of numbers representing rolls on a die.
  * @param {number} value - A specific value to find.
  * @returns {*} - The found value or `null`.
  */
-function findValue(rolls, value) {}
+function findValue(rolls, value) {
+  // method 1
+let flag = null           //accumulator
+  if (rolls.length === 0) // this is to check if rolls is [] but no need because for loop executes and nothing it will automatically return empty []
+      flag = null
+  else if (rolls.includes(value))
+      flag =  value
+  else
```

Comment:
since `flag` was initiated with the value `null` you don't need this extra else statement.